common/web/gesture-recognizer/src/engine/headless/touchpointCoordinator.ts

```diff
@@ -223,7 +223,21 @@ export class TouchpointCoordinator<HoveredItemType, StateToken=any> extends Even
     const selector = this.currentSelector;
 
     touchpoint.setGestureMatchInspector(this.buildGestureMatchInspector(selector));
-    this.emit('inputstart', touchpoint);
+
+    // If there's an error in code receiving this event, we must not let that break the flow of
+    // event input processing here!
+    try {
+      this.emit('inputstart', touchpoint);
+    } catch (err) {
+      console.error(err);
+    }
+
+    // In particular, things will break HORRIBLY if this code block does not get to run.
```

Comment:
This comment is a justification for adding the try/catch above. It's really about the _change_ you are making now and not terribly helpful for understanding the code flow.

What does HORRIBLY even mean. Can you be specific about the kinds of problems that arise?